javascript-loops-practice/filter-out-nulls.js

```diff
@@ -0,0 +1,13 @@
+/* exported filterOutNulls */
+function filterOutNulls(values) {
+  const arr = [];
+
+  for (let i = 0; i <= values.length; i++) {
+    if (values[i] === null) {
+      i++;
+    } else {
+      arr.push(values[i]);
```

Comment:
I don't believe you need the `i++` on line 7. The loop will automatically increase the iterator after every iteration because of the last statement on line 5. 

Also this is hard to read, it'd be easier if you flip the logic to `!==` and only push the result if that case matches. Then you don't need an `else` statement.